.github/CONTRIBUTING.md

```diff
@@ -129,26 +129,28 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
   New changes always go into the **Unreleased** section.
   Keeping the changelog up-to-date simplifies the release process for Maintainers.
   An example:
-    ```
-    Unreleased
-    ----------
+  ```
+  Unreleased
+  ----------
 
-    * `Update-Item` now supports `-FriendlyName`.
-    ```
-    Please use the present tense and imperative mood when describing your changes:
-
-      * Instead of "Adding support for Windows Server 2012 R2", write "Add support for Windows Server 2012 R2".
-
-      * Instead of "Fixed for server connection issue", write "Fix server connection issue".
+  * `Update-Item` now supports `-FriendlyName`.
```

Comment:
Better to include the PR number as well, like `Update-Item now supports -FriendlyName (#1234)`